sway-core/src/type_system/type_engine.rs

```diff
@@ -533,6 +537,10 @@ pub fn insert_type(ty: TypeInfo) -> TypeId {
     TYPE_ENGINE.insert_type(ty)
 }
 
+pub fn type_engine_size() -> usize {
```

Comment:
What is the reason for adding this function?

Reply:
No real reason, I've been working adding improvements to the type system and reducing the complexity / number of `TypeId`'s created, etc. This is just a helper function to help debug with that---just figure'd I'd leave it in there. I can remove it if you'd prefer.

Reply:
Thanks! No need to remove it :)